app/views/admin/booths/index.html.haml

```diff
@@ -2,7 +2,7 @@
   .col-md-12
     .page-header
       %h1
-        Booths
+        #{t'Booth'}s
```

Comment:
Let's introduce the plural in the translation file please

Reply:
alright!

Reply:
I18n supports plurals, please check: https://guides.rubyonrails.org/i18n.html